### PR TITLE
Create welcome.md

### PR DESCRIPTION
**Pull request by caradelia**

### CODE REVIEW

File: website/content/about/welcome.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
```
Comment (leonardpahlke) [inline website/content/about/welcome.md:1]:
```suggestion
---
title: Welcome to the CNCF Environmental Sustainability Technical Advisory Group (TAG)
description: The CNCF Environmental Sustainability Technical Advisory Group (TAG) is a community of experts focused on promoting sustainable practices and reducing the environmental impact of cloud native technologies.
---

```
We are missing the header here (feel free to change the title and description) :)

**Review by leonardpahlke — COMMENTED** [review]
Thanks @caradelia! The page does not appear on the website (see link posted by the bot) I think we need to add the header (see proposed change). 

Also, this feels a bit like a replica of the "main page", thoughts? Maybe just extend 
[`website/content/_index.md`](https://github.com/cncf/tag-env-sustainability/blob/main/website/content/_index.md)?

File: website/content/about/welcome.md (hunk 1 above)
Comment (caradelia) [reply website/content/about/welcome.md:1]: Yes, I agree on the duplication. I was playing around with. Let me fix.

File: website/content/about/welcome.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+The [TAG Environmental Sustainability Charter](charter.md) outlines the scope of the group activities as well as the goals and intended deliverables.
+
+## Join us 
+
+* Meetings: every 1st and 3rd Wednesday at 5pm CEST/GMT+2 (8am Pacific) on [Zoom](https://zoom.us/my/cncftagenvsustainability)
+* [Meeting notes & agenda](https://docs.google.com/document/d/1TkmMyXJABC66NfYmivnh7z8Y_vpq9f9foaOuDVQS_Lo/edit#)
+* [Mailing List](https://lists.cncf.io/g/cncf-tag-env-sustainability/) 
+* [Slack](https://cloud-native.slack.com/archives/C03F270PDU6) #tag-environmental-sustainability
+* [Github](https://github.com/cncf/tag-env-sustainability)
+
```
Comment (catblade) [inline website/content/about/welcome.md:19]: Add twitter maybe?  We need to get posting updated invites there too.

**Comment by leonardpahlke** [comment]
I opened up an issue recently https://github.com/cncf/tag-env-sustainability/issues/146 could we update this PR to resolve this issue maybe?

**Pull request closed without merging**